### PR TITLE
New Nuxt Static Landing Page

### PR DESCRIPTION
**Pull request by YacineMessaadi**

### CODE REVIEW

File: .vscode/extensions.json [hunk 1]
```diff
@@ -1,11 +0,0 @@
-{
-  "recommendations": [
```
Comment (riderx) [inline .vscode/extensions.json:2]: this should be adapted not removed

File: .github/workflows/test.yml [hunk 2]
```diff
@@ -1,56 +0,0 @@
-name: Test
```
Comment (riderx) [inline .github/workflows/test.yml:1]: this should be adapted not removed

File: .forestry/settings.yml [hunk 3]
```diff
@@ -1,26 +0,0 @@
----
```
Comment (riderx) [inline .forestry/settings.yml:1]: this should be adapted not removed

File: .eslintrc [hunk 4]
```diff
@@ -1,3 +0,0 @@
-{
-  "extends": "@antfu"
```
Comment (riderx) [inline .eslintrc:2]: this should be adapted, not removed maybe check https://github.com/antfu/vitesse-nuxt3

Comment (riderx) [reply .eslintrc:2]: Vitesse was the base i used for both websites

File: LICENSE [hunk 5]
```diff
@@ -1,21 +0,0 @@
-MIT License
-
-Copyright (c) 2020-2021 Martin Donadieu
```
Comment (riderx) [inline LICENSE:3]: This should stay

Comment (riderx) [reply LICENSE:3]: maybe we have to check for blog if different license should apply

File: content/blog/amrap.md [hunk 6]
```diff
@@ -1,12 +1,15 @@
...
+slug: "amrap"
+title: "AMRAP Guide: What are the benefits - 2022"
+surtitle: ""
+subtitle: ""
+published: true
 description: AMRAP stand for “As Many Reps/Rounds As Possible.” This workout involves you competing with the clock,  complete as many reps or rounds as you can in a given time.
 author: Martin Donadieu
 date: 2022-01-12
-head_image: "/amrap1.webp"
-head_image_alt: Crossfit Men doing sport and thinking about AMRAP
```
Comment (riderx) [inline content/blog/amrap.md:7]: Why is the change not full camel case then ?

File: .forestry/settings.yml (hunk 3 above)
Comment (riderx) [reply .forestry/settings.yml:1]: We use forestry to manage content for non tech people ( copywriting freelance)

File: content/blog/alternative-to.md [hunk 7]
```diff
@@ -36,8 +37,8 @@ Rogue echo gym trainer is considered a gold standard in fitness circles, and for
 
 But since Rogue Echo Gym Timer comes with specific hardware, you have to spend a fortune to have it set in your fitness training area. But does that mean you have got no options other than this timer? Well, no. [Captime](http://onelink.to/captime) is one of the best alternatives to the Rogue Echo gym Timer.
 
-[Captime](http://onelink.to/captime) is a mobile app that you can install on your iPhone or iPad. 
-<Cta/>
```
Comment (riderx) [inline content/blog/alternative-to.md:40]: Call to action is not in place in blog article ? where it is now ?

File: content/draft/c-est-quoi-indie-makers.md [hunk 8]
```diff
@@ -0,0 +1,68 @@
+---
+slug: c_est_quoi_indie_makers
```
Comment (riderx) [inline content/draft/c-est-quoi-indie-makers.md:2]: that shouldn't be there

File: content/draft/comment-trouver-un-associe.md [hunk 9]
```diff
@@ -0,0 +1,118 @@
+---
+slug: comment-trouver-un-associe
```
Comment (riderx) [inline content/draft/comment-trouver-un-associe.md:2]: same ^^

File: locales/de.yml [hunk 10]
```diff
@@ -1,14 +0,0 @@
-button:
-  about: Über
```
Comment (riderx) [inline locales/de.yml:2]: translation was not used but will be in the future, do we still have it?

File: src/pages/blog/farmer-s-walk-benefits-best-exercise-to-strengthen-your-core-muscles.md [hunk 11]
```diff
@@ -1,49 +0,0 @@
----
-title: 'Farmer’s Walk Benefits: Best Exercise To Strengthen Your Core Muscles'
```
Comment (riderx) [inline src/pages/blog/farmer-s-walk-benefits-best-exercise-to-strengthen-your-core-muscles.md:2]: a forgot one ?

File: src/pages/blog/how-to-perform-murph-workout-and-variations-to-try.md [hunk 12]
```diff
@@ -1,89 +0,0 @@
----
-title: How To Perform Murph Workout and Variations To Try
-description: Murph workout is considered to be cruel in its simplicity, absolutely
```
Comment (riderx) [inline src/pages/blog/how-to-perform-murph-workout-and-variations-to-try.md:3]: same?

File: windi.config.js [hunk 13]
```diff
@@ -0,0 +1,37 @@
+module.exports = {
+  attributify: true,
+  plugins: [
```
Comment (riderx) [inline windi.config.js:3]: still using windi ?

Comment (riderx) [reply windi.config.js:3]: if yes why the config differ ?

**Review by riderx — CHANGES_REQUESTED** [review]

File: .eslintrc (hunk 4 above)
Comment (YacineMessaadi) [reply .eslintrc:2]: Vite is supported by default in Nuxt 3, the Antfu's template just bring UnoCSS support

File: content/blog/alternative-to.md (hunk 7 above)
Comment (YacineMessaadi) [reply content/blog/alternative-to.md:40]: It is implemented directly in the blog/[id].vue file

**Pull request merged**